### ISSUE TITLE
Moving between pages some screen readers focus on the wrong element

### ISSUE THREAD

**Issue by rmccar** [ONSdigital/design-system#3135]
In the latest eQ DAC report there is an issue with some screen readers focusing on the wrong element when moving between pages. On the new page Instead focus begins near the top of the main content, either on the ‘Previous’ link, or on the heading level 1 contained within the main region. This could be disorienting for these screen reader users, who may take more time to determine their location when moving between each page. This issue was not consistent across different devices. I have been unable to recreate this using VoiceOver on macOS. The focus is always set to the skip to content link at the top of the page.

DAC issue: DAC_Screen_Reader_Focus_Usability_01

**Comment by daniellecorke** [comment]
Can we get access to all the screen readers required?

**Comment by rmccar** [comment]
Got clarification from DAC on this, this is specific to the voiceover on iOS and JAWS on Windows which is why I was unable to recreate this on macOS. They said that "Nothing fundamental in the code appeared to make focus jump, which suggests this behaviour may be intermittent depending on the assistive technologies, and may be more the fault of the screen reader, but we still report these issues to make clients aware." This plus the fact this is only a usability and not an actual A graded accessibility fail makes  me think we can just close this.
